SonarQube.Client.Tests/Models/ServerSentEvents/SSEStreamFactoryTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.IO;
+using System.Threading;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarQube.Client.Models.ServerSentEvents;
+
+namespace SonarQube.Client.Tests.Models.ServerSentEvents
+{
+    [TestClass]
+    public class SSEStreamFactoryTests
+    {
+        [TestMethod]
+        public void Create_CreatesSSEStream()
```

Comment:
Pretty useless test ngl, but it doesn't hurt to have it

Reply:
Yeah it's pretty much just for the code coverage...